frontend/src/app/register/page.tsx

```
@@ -11,105 +11,113 @@ const kelly = Kelly_Slab({
-                Already have an account yet?
-                <Link href="/login" className="font-bold pl-1 md:pl-3 underline">
-                  Login Now
-                </Link>
-              </p>
-            </form>
-          </div>
+                <div className="flex flex-col md:flex-row md:gap-3">
+                  <div className="">
+                    <label className="label">
+                      <span className="label-text text-xs md:text-sm">Password</span>
+                    </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)